java/java.editor/src/org/netbeans/modules/java/editor/rename/InstantRenamePerformer.java

```diff
@@ -450,14 +450,13 @@ private static boolean allowInstantRename(CompilationInfo info, Element e, Eleme
             if(isProperty) {
                 return false;
             }
-            boolean isRecordComponent = e.getEnclosingElement().getKind() == ElementKind.RECORD;
-            if (isRecordComponent) {
-                return false;
-            }
```

Comment:
might be still needed after all?

Reply:
I just placed the new check at a wrong place. Sorry for that.